networkit/graph.pyx

```diff
@@ -394,6 +394,15 @@ cdef class Graph:
 		"""
 		self._this.sortEdges()
 
+	def sortEdgesBy(self, object callback):
+		cdef SortEdgesLambdaWrapper* wrapper
+		try:
+			wrapper = new SortEdgesLambdaWrapper(callback)
+			with nogil:
+				self._this.sortEdges[SortEdgesLambdaWrapper](dereference(wrapper))
+		finally:
+			del wrapper
+
```

Comment:
Is there any test for this method? As this may be a bit fragile I would suggest to add a test case for it.

Reply:
This was a first attempt to provide `sortEdgesByWeight` to Python, which is now implemented in `nk.graphtools`. Since it is not needed anymore I think it is a better idea to keep it for C++ only.